### DISCUSSION TITLE
Requesting for better AltStore UTM and UTM SE description

### DISCUSSION THREAD

**Discussion by bachig26** [utmapp/UTM#3994] [General]
The current AltStore version of UTM and UTM SE has same description and there is differentiation in any case.

I would like to request for better description and provide clear information between them.

In case as many know (to view in browser), AlStore Browse => https://altsource.by.lao.sb/browse/?source=https%3A%2F%2Falt.getutm.app

**Comment by osy** [comment]
What specifically do you need to know? The difference is one has JIT and one does not have JIT. It’s mentioned in the description.

**Reply by bachig26** [reply]
From my point of view, if I came directly to the AltStore without knowledge of UTM and the GitHub repo, one needs to search through the internet and spend looking for the understanding of the both version.

The discussion I opened is for asking for an possibility of better explanatory description to the AltStore UTM repo.

I can see in the description,
UTM => _This version only runs on < iOS 14.0 and iOS 14.2-14.4 (A14+ devices only). All other devices should install UTM SE._

UTM SE => _This version runs on all > iOS 11 devices but lacks JIT support and is slower than regular UTM._

_As it is just one line at the bottom, most may miss it because of the 90+% same texts in both version, that's how I missed it and confused between them._

**Reply by osy** [reply]
Okay, I’ll add something like that.
